myst/src/utils/config.rs

```diff
@@ -103,7 +103,7 @@ impl Config {
             aws_secret: config.get_str("aws_secret").unwrap().to_string(),
             temp_data_path: config.get_str("temp_data_path").unwrap().to_string(),
             download_frequency: config.get_int("download_frequency").unwrap() as u64,
-            plugin_path: config.get_str("plugin_path").unwrap(),
+            plugin_path: config.get_str("plugin_path").unwrap_or(String::from("/etc/myst/plugin/metrics-reporter")),
```

Comment:
plugin modules may typically come from /var or /opt

Reply:
how about having all myst related binaries and plugins in `/usr/share/myst` , conf in `/etc/myst`  and data and logs in `/var/myst`